platform/darwin/src/MGLPointCollection.h

```diff
@@ -0,0 +1,44 @@
+#import <Foundation/Foundation.h>
+#import <CoreLocation/CoreLocation.h>
+
+#import "MGLOverlay.h"
+#import "MGLShape.h"
+
+/**
+ The `MGLPointCollection` class is used to define shapes composed of multiple 
+ points.
```

Comment:
@1ec5 I'm thinking about mentioning that this class is really only useful as an `MGLPointCollectionFeature` in an `MGLGeoJSONSource`. What do you think?